app/views/stations/edit.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Edit Station Info</h1>
+
+<form action='/stations/<%= @station.id %>' method = "POST">
+  <input type="hidden" name="_method" value="PUT" />
+  <label for="">Station Name</label>
+  <input type="text" name="station[name]" value="<%= @station.name %>" />
+  <label for="">City</label>
+  <input type="text" name="city[name]" value="<%= @station.city.name %>"/>
+  <label for="">Dock Count</label>
+  <input type="number" name="dock_count[dock_number]" value="<%= @station.dock_count.dock_number %>"/>
+  <label for="">Installation Date</label>
+  <input type="date" name="installation_date[installation_date]" value="<%= @station.installation_date.installation_date %>"/>
```

Comment:
Having a value named the same as the table usually give us some weird syntax like this `installation_date.instalation_date`... ECHO.... I think Sal mentioned above to rethink this schema a little which will help with the naming.